cpp/src/arrow/compute/kernels/scalar_string_test.cc

```diff
@@ -2271,7 +2279,7 @@ TYPED_TEST(TestStringKernels, UTF8ZeroFill) {
                    R"(["xxx1", "-xx2", "+xx3"])", &options);
 
   // Non-ASCII padding character
-  options = ZeroFillOptions{/*width=*/5, u8"💠"};
+  options = ZeroFillOptions{/*width=*/5, "💠"s};
```

Comment:
It wasn't necessary to add the literal, was it? A plain C string would probably have worked: `"💠"`

Reply:
You're right...
I was not a smart...